### PR TITLE
LRDOCS-7691 Setting Up Liferay

### PR DESCRIPTION
**Pull request by jhinkey**
https://issues.liferay.com/browse/LRDOCS-7691

- Added the checklist table
- Added all leading checklist Setup Tasks to the TOC
- Moved the content to setting_up_liferay.rst
- Renamed the section folder to setting-up-liferay
- Added Setting Up Liferay to the landing page's Installation and Upgrades card

Because of the folder name change and because Setting Up Liferay is referenced from many places, we'll want to merge this soon. :-)

### CODE REVIEW

File: docs/dxp/7.x/en/installation-and-upgrades/setting_up_liferay.rst [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+Configure Users
+~~~~~~~~~~~~~~~
+
+After you `understand Users <../users-and-permissions/users/understanding-users.md>`_, configure them for your instance.
+
+* Enable/disable User fields
+* Add any necessary `custom User fields <../users-and-permissions/devops/adding-custom-fields-to-users.md>`_
+* Specify whether to allow strangers to create accounts
+* Define `Users authentication <../system-administration/configuring-liferay/virtual-instances/user-authentication.md>`_
```
Comment (jrhoun) [inline docs/dxp/7.x/en/installation-and-upgrades/setting_up_liferay.rst:85]: > * Define `Users authentication <../system-administration/configuring-liferay/virtual-instances/user-authentication.md>`_

Which link is better, the one to user auth in virtual instances? OR, https://learn.liferay.com/dxp/7.x/en/installation-and-upgrades/securing-liferay/authentication-basics.html ?

File: docs/dxp/7.x/en/installation-and-upgrades/setting_up_liferay.rst [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+* Configure automatic association of Users to specific Sites, `Roles </users-and-permissions/roles-and-permissions/understanding-roles-and-permissions.>`_, and `User Groups <../users-and-permissions/user_groups.html>`_
```
Comment (jrhoun) [inline docs/dxp/7.x/en/installation-and-upgrades/setting_up_liferay.rst:86]: > * Configure automatic association of Users to specific Sites, `Roles </users-and-permissions/roles-and-permissions/**understanding-roles-and-permissions.**>`_, and `User Groups <../users-and-permissions/user_groups.html>`_

missing `.md` extension

Comment (jhinkey) [reply docs/dxp/7.x/en/installation-and-upgrades/setting_up_liferay.rst:86]: Good catch. Sorry about that. Fixed them.

**Comment by jrhoun** [comment]
Merged. Thanks @jhinkey

**Pull request closed without merging**